src/System.ServiceModel.Primitives/tests/ServiceModel/SynchronizedCollection.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+using Infrastructure.Common;
+using Xunit;
+
+public class SynchronizedCollectionTest
+{
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
+#endif
+    [WcfFact]
+    // This Unit test is ported from full framework test
+    public static void SynchronizedCollectionPublicMembersTest()
+    {
+        SynchronizedCollection<int> coll = new SynchronizedCollection<int>();
```

Comment:
Could you run an InnerLoop test with /p:Coverage=true and see what code in SynchronizedCollection is not being covered?   At a glance, I see we're missing the different ctors.  Not blocking, but as long as we're adding this test, this is the right time to do it.  We don't need to go overboard, just ensure every public entry point is at least called.